modules/gsqlite3backend/gsqlite3backend.cc

```diff
@@ -77,6 +77,7 @@ class gSQLite3Factory : public BackendFactory
     declare(suffix, "database", "Filename of the SQLite3 database", "powerdns.sqlite");
     declare(suffix, "pragma-synchronous", "Set this to 0 for blazing speed", "");
     declare(suffix, "pragma-foreign-keys", "Enable foreign key constraints", "no" );
+    declare(suffix, "journal-mode", "SQLite3 journal mode", "WAL");
```

Comment:
Why journal-mode instead of pragma-journal-mode, like the settings above this one?

Reply:
Good point!

Reply:
pushed